core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3903,7 +3903,7 @@ public synchronized HttpRedirect doQuietDown() {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
-     * @deprecated since TODO; use {@link #doQuietDown(boolean, int, String)} instead.
+     * @deprecated since 2.267; use {@link #doQuietDown(boolean, int, String)} instead.
```

Comment:
We don't generally note when something was deprecated, but it was in the original so I just filled in the version.